components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -185,16 +175,11 @@ net::NetworkTrafficAnnotationTag GetNetworkTrafficAnnotationTag() {
     )");
 }
 
-void RegisterResourceComponentsForCountryCode(const std::string& country_code) {
-  g_brave_browser_process->resource_component()
-      ->RegisterComponentForCountryCode(country_code);
-}
-
-void RegisterResourceComponentsForDefaultLanguageCode() {
+void RegisterResourceComponentsForDefaultLanguageCode(
```

Comment:
Maybe a member function as then we will not need to pass `resource_component_`?

Reply:
Fixed